tests/foreman/ui/test_host.py

```diff
@@ -409,7 +410,13 @@ def test_positive_read_from_edit_page(session, host_ui_options):
 
 
 def test_positive_assign_taxonomies(
-    session, module_org, smart_proxy_location, target_sat, function_org, function_location_with_org
+    module_org,
+    smart_proxy_location,
+    target_sat,
+    function_org,
+    function_location_with_org,
```

Comment:
The test is using both `module_org` and `function_org`, could you update it to use only one?
Also, we have a lot of fixture scope mismatch in the tests, which can lead to unwanted test results.
Tests are using `target_sat` while a lot of fixtures are using `module_target_sat` internally.
I filed https://github.com/SatelliteQE/robottelo/issues/20456 for it; you can address it separately.